nextjs.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  root: true,
+  extends: [
+    'next/core-web-vitals',
+    './react', // Keep next-to-last: ensures our TS overrides have priority
+  ],
+};
```

Comment:
Could you add an empty `rules` object with a comment so that it will be clear where next-js specific rule deviations should go?

Also, this file needs to be indexed for NPM in `package.json`: Add `"nextjs.js"` to the `files` array. Otherwise it won't be included in the published bundle :-)